Xamarin.Forms.Platform.UAP/Platform.cs

```diff
@@ -23,8 +23,6 @@ public abstract class Platform : INavigation
 	{
 		static Task<bool> s_currentAlert;
 
-		internal static StatusBar MobileStatusBar => ApiInformation.IsTypePresent("Windows.UI.ViewManagement.StatusBar") ? StatusBar.GetForCurrentView() : null;
```

Comment:
These changes were made  to make the https://www.nuget.org/packages/Microsoft.Toolkit.Uwp.PlatformSpecificAnalyzer happy. Though there appears to a bug with that analyzer currently that's causing it to throw an NRE which is why it's not installed